### ISSUE TITLE
GRN inference

### ISSUE THREAD

**Issue by karimelbarbary16** [oubounyt/SCANet#8]
Hi,

thank you for this nice tool. I followed the tutorial without any issues appearing until excuting this cell : 

grn_df = sn.grn.grn_inference(adata_processed=adata_processed,modules_df=modules_df, module=Mod_, groupby_=groupby_, anno_name=anno_name, specie=specie_, subsampling_pct=80, n_iteration=n_iteration, num_workers=num_workers)

I always get the response "0 edges where found". I tried to change the number of workers and iterations but every time I get the same response. After some debugging, I found out that the problem is in this line No. 261 in grn.py file "result_list = pool.map(GRN.regulon, args_)". The function regulon is never excuted by me. Could you please help resolving this matter? Thanks !

**Comment by oubounyt** [comment]
Certainly! however without further information about your specific setup and data, it's challenging to pinpoint the exact cause of the problem.
The error message "0 edges were found" could indicate various issues, such as missing transcription factors, incorrect ranking databases, or species mismatches. To assist you better, could you provide more details about your environment, the data you're working with, and any relevant parameters you're using in your function calls? This additional information will help me offer more targeted assistance. Thanks!

**Comment by karimelbarbary16** [comment]
Thanks for your prompt reply! Actually I am working on the example you provided : https://github.com/oubounyt/SCANet/blob/main/docs/full-example.ipynb

I am using exactly the same parameters. I reduced only the number of workers to 40 instead of 60 and the number of processes to 6 instaed of 10 in line 261 in grn.py file. The human database was successfully downloaded in the databases folder. Thanks!
